### ISSUE TITLE
When will you release your training and testing data?

### ISSUE THREAD

**Issue by JiaxiongQ** [dlichy/ShapeAndMaterial#4]
Thanks for your great work!

**Comment by dlichy** [comment]
Thanks for your interest. I'm planning on uploading the dataset in one to two weeks.

**Comment by JiaxiongQ** [comment]
Ok, thank you!

**Comment by DESMONDLUCY** [comment]
this work is awesome，I want to try it by myeself now.

**Comment by DESMONDLUCY** [comment]
Will it take much longer to wait for test set data

**Comment by dlichy** [comment]
I have just uploaded the training and testing data.
[real test data](https://github.com/dlichy/ShapeAndMaterial#download-our-real-dataset)
[synthetic training data](https://github.com/dlichy/ShapeAndMaterial#download-the-training-data)

Thank you for your interest in our project.
